python/orca/src/bigdl/orca/data/file.py

```diff
@@ -578,3 +577,51 @@ def fs_load(path, *args, **kwargs):
                     os.remove(temp_path)
 
     return fs_load
+
+
+def enable_multi_fs_save(save_func):
+
+    @functools.wraps(save_func)
+    def fs_save(obj, path, *args, **kwargs):
+        from bigdl.dllib.utils.file_utils import is_local_path
+        if is_local_path(path):
+            return save_func(obj, path, *args, **kwargs)
+        else:
+            import uuid
+            import tempfile
+            from bigdl.dllib.utils.file_utils import append_suffix
+            file_name = str(uuid.uuid1())
+            file_name = append_suffix(file_name, path)
+            temp_path = os.path.join(tempfile.gettempdir(), file_name)
+
+            try:
+                result = save_func(obj, temp_path, *args, **kwargs)
+                put_local_file_to_remote(temp_path, path)
```

Comment:
Do we need to consider whether we can overwrite the original content?

Reply:
the function only supports filemode, no overwrite option, pending this for now.